frontend/src/components/App.jsx

```diff
@@ -7,6 +7,13 @@ import Train from 'containers/Train';
 import Search from 'containers/Search';
 
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Doesn't look like this state is used at all.

Reply:
Yep, it's been removed.